cmd/minishift/cmd/delete.go

```diff
@@ -40,8 +40,6 @@ func runDelete(cmd *cobra.Command, args []string) {
 	api := libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))
 	defer api.Close()
 
-	setSubcriptionManagerParameters()
-
```

Comment:
This we are removing because it's conflict with start.go variables and viper not able to make out what to do.